test/aqs_test_module.yaml

```diff
@@ -0,0 +1,96 @@
+# Simple test spec
```

Comment:
@gwicke see `/projects/aqs_default.yaml` it sets up an example prod aqs. These are just fake endpoints to enter the test data into cassandra.